.github/actions/pkg-check/action.yml

```diff
@@ -5,6 +5,7 @@ inputs:
   pkg-name:
     description: package name inside lightning.*
     required: true
+    default: ""
```

Comment:
should default be lightning?

Reply:
it can be anything except `app|lite|pytorch` so why add noise to have defaulted, in fact, default is empty as most users shall install straight without setting any env. variables